config/README.md

```diff
@@ -1,6 +1,67 @@
-We use [Firebase Remote Config](https://firebase.google.com/docs/remote-config) for dynamic configuration of our app.
+This project is based on [REST APIs using the node client](https://github.com/firebase/quickstart-nodejs/tree/master/config) to manipulate remote config in firebase.
+
+More information about what we are using remote config for can be found in this [private gdrive document](https://docs.google.com/document/d/1UNAAbFE_U8KVSd5SNg_Z0Gm9Kn7TY2UbQbbsDdz6ZGg)
+
+The project is written using our favourite command / programming languages `javascript` and `bash`, though fear not, the project can easily be converted to other languages, here are some other quickstart examples of firebase remote config:
+- https://github.com/firebase/quickstart-java/tree/master/config
+- https://github.com/firebase/quickstart-python/tree/master/config
+
+### Environment
+
+```
+node -v
+v10.15.3
+npm -v
+6.11.3
+```
+
+### Prerequisites
+1. Download the corresponding service account files and add them to .secrets
```

Comment:
.secrets where? Root directory of the project, your user, or what? Be specific.

Reply:
I've added a step describing that you need to create a folder .secrets in the current folder.